docs/src/main/sphinx/release/release-376.md

```diff
@@ -0,0 +1,83 @@
+# Release 376 (7 Apr 2022)
+
+## General
+
+* Add table redirection awareness for `RENAME table` operations. ({issue}`11277`)
+* Deny adding column with comment if the connector does not support this feature. ({issue}`11486`)
+* Improve query plans and performance in the presence of inequality expressions. ({issue}`11518`)
+* Use "is distinct" semantics in `array_except`, `array_union`, `map_concat`,
+  `map_from_entries`, `multimap_from_entries`, and `multimap_agg`. ({issue}`560`)
+* Fix operator type exception that occurred when index join was used during
+  query execution. ({issue}`11758`)
+* Fix stage memory statistics reporting for queries running with `retry-policy`
```

Comment:
What was broken? Incorrect values? Failures?

Reply:
From the PR 

Without the fix stage level statistics for peak and current memory were off when task retries happened.

Not sure what is user facing of that..

Reply:
Ok, then let's rephrase as:

> Fix incorrect stage memory reporting for queries ....